index.php

```diff
@@ -23,4 +23,96 @@
 //
 //
 //echo mt_rand(1, 15). "<br>";//рандомной число включительно
-//echo min(1, 2, 3) ;// минимальное из чисел
\ No newline at end of file
+//echo min(1, 2, 3) ;// минимальное из чисел
+//...............................................СТРОКОВЫЕ ОПЕПАЦИИ.............................
+
+/*$str = 'Hello world';
+echo "VAR : $str";
+
+$length = strlen($str);
+echo trim('   some  ');//вывод без пробелов в начале и конце слова
+$upper = strtoupper(' some');//слово верхний регистр
+$lower = strtolower('soMe');//слово в нижний регистр
+echo strtoupper(trim( ' some   '));//обьединение ф-ций
+echo md5('qwerty');//Хеширование текста*/
+
+//...............................................УСЛОВНЫЕ ОПЕРАТОРЫ.......................................
+
+/*$a = 5;
+if($a == 5){//условия ==, !=, >, <, <=, >=
+    echo "YES";
+}
+
+$str = 'Hello';
+if($str == 'Hello'){//условия ==, != для STRING
+    echo "YES";
+} else {
+    echo 'NO';
+}
+
+$a = 4;
+$str1 = 'HEllo';
+
+if($str1 = 'Hello'){
+    echo 'yes';
+}elseif ($a == 4){//использование ifelse
+    echo 'yes';
+}else{
+    echo 'no';
+}
+
+$a = 5;
+$str2 = 'hello';
+$isWeatherGood = false;
+
+if(str2 != 'HEllo' || !$isWeatherGood){//!isWeatherGood == isWeatherGood = false// || - ИЛИ, && - И
+    echo "FIRST";
+}elseif($a == 5){
+    $res = 'SECOND';
+    echo $res;
+}else{
+    echo 'no';
+}*/
+//.....................................ОПЕРАТОР....SWITCH CASE............................................
+/*
+$x = 6;
+
+switch ($x){
+    case 5:
+        echo "Var: 5";
+        break;
+    case 6:
+        echo "Var: 6";
+        break;
+    case 7:
+        echo "Var: 7";
+        break;
+    default:
+        echo 'NO RESULT';
+        break;
+}*/
+//.............................................МАССИВЫ..............................................
+
+//$nums = array(12,3,4,5,5,6,4);//массив
+//$num1 = [12,3,4,5,5,6,4];//одинаковый массив по разному обьявлен
+//$nums [1] = 10;//в скобках указываем индекс и переопределяем
```

Comment:
please follow single code format for the whole code, I mean space between array name and index.
Looks like it's better to write like this:   $arr[i] (without space)